exporters/otlp/otlpmetric/otlpmetricgrpc/readme.md

```diff
@@ -0,0 +1,48 @@
+# OpenTelemetry-Go OTLP Metric GRPC Exporter
+
+[![Go Reference](https://pkg.go.dev/badge/go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetricgrpc.svg)](https://pkg.go.dev/go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetricgrpc)
+
+[OpenTelemetry Protocol Exporter](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.20.0/specification/protocol/exporter.md) implementation.
+
+## Installation
+
+```
+go get -u go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetricgrpc
+```
+
+## Examples
+
+- [HTTP Exporter setup and examples](./example_test.go)
+
+## [`otlpmetricgrpc`](https://pkg.go.dev/go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetricgrpc)
+
+Exporters should be created using the [New] and used with a [PeriodicReader].
```

Comment:
Hyperlink for `New` is missing.

Reply:
The PeriodicReader isn't exported in the latest version, just on main.  So if I add a link to it right now, the check-links CI job will fail until a new version is released.

Reply:
Then maybe just `PeriodicReader` and `New` instead of hyperlinks 😬